onto-viewer-core/src/main/java/org/edmcouncil/spec/ontoviewer/core/ontology/updater/UpdaterThread.java

```diff
@@ -31,187 +31,185 @@
-      // TODO: We take default labels only to put them in label provider a few lines down...
+        }
+        try {
+            if (isInterrupt()) {
+                throw new InterruptUpdate();
+            }
+
+            job = UpdaterOperation.setJobStatusToInProgress(job);
+
+            OWLOntology ontology = null;
+            String msgError = null;
+
+            LOG.info("Configuration loaded ? : {}", config != null
+                    || !config.getCoreConfiguration().isEmpty());
+            LOG.info("File system manager created ? : {}", fileSystemManager != null);
+
+            CoreConfiguration viewerCoreConfiguration = config.getCoreConfiguration();
+
+            if (isInterrupt()) {
+                throw new InterruptUpdate();
+            }
+
+            //download ontology file/files
+            //load ontology to var
+            try {
+                AutoOntologyLoader loader = new AutoOntologyLoader(fileSystemManager, viewerCoreConfiguration);
+                ontology = loader.load();
+            } catch (OWLOntologyCreationException ex) {
+                msgError = ex.getMessage();
+                LOG.error("[ERROR]: Error when creating ontology. Stoping application. Exception: {} \n Message: {}", ex.getStackTrace(), ex.getMessage());
+            } catch (IOException ex) {
+                msgError = ex.getMessage();
+                LOG.error("[ERROR]: Cannot load ontology. Stoping application. Stack Trace: {}", Arrays.toString(ex.getStackTrace()));
+            } catch (ParserConfigurationException ex) {
+                msgError = ex.getMessage();
+                LOG.error("[ERROR]: Cannot load ontology, parser exception. Stoping application. Stack Trace: {}", Arrays.toString(ex.getStackTrace()));
+            } catch (XPathExpressionException ex) {
+                msgError = ex.getMessage();
+                LOG.error("[ERROR]: Cannot load ontology, xpath expression exception. Stoping application. Stack Trace: {}", Arrays.toString(ex.getStackTrace()));
+            } catch (SAXException ex) {
+                msgError = ex.getMessage();
+                LOG.error("[ERROR]: Cannot load ontology, sax exception. Stoping application. Stack Trace: {}", Arrays.toString(ex.getStackTrace()));
+            }
+
+            if (msgError != null) {
+                LOG.error("[ERROR]: Cannot update, id {}", job.getId());
+                job = UpdaterOperation.setJobStatusToError(job, msgError);
+                blocker.setUpdateNow(Boolean.FALSE);
+                return;
+            }
+            if (isInterrupt()) {
+                throw new InterruptUpdate();
+            }
+
+            Set<String> scopes = scopeIriOntology.getScopeIri(ontology);
+
+            //get default labels
+            // TODO: We take default labels only to put them in label provider a few lines down...
 //      Map<String, String> defaultLabels = labelProvider.getDefaultLabels();
+            //get default text searcher db
```

Comment:
why is it commented out? not all default labels are given in the file, what was the reason? 
@trojczak @patrycjamia

Reply:
I think I've commented it out because the only usage of `defaultLabels` was on line 162 where we put it back where they were taken from on #145. So it seemed to me as some kind of code that was left after other changes and now it is not needed anymore.